README.md

```diff
@@ -4,6 +4,55 @@
 
 AtlasGenerator is a Spark Job that generates [Atlas](https://github.com/osmlab/atlas) shards from OSM pbf shards (built from an OSM database with osmosis).
 
+## Getting started
+
+It is recommended to use Oracle's latest JDK 1.8 with that project.
+
+### Build
+
+This command will create the necessary fat (contains all the dependencies) and shaded (contains all the dependencies, except spark and hadoop) jars.
+
+```
+gradle clean shaded fat
+```
+
+### Setup example: BLZ (Belize)
+
+Download the country boundaries and the sharding tree files from the respective sub-folders available [here](https://apple.box.com/s/3k3wcc0lq1fhqgozxr4mdi0llf95byo3). We will then call `file:///path/to/boundaries/world_boundaries_osm_20170424.txt` and `file:///path/to/sharding/tree-6-14-100000.txt` the boundaries and sharding files just downloaded. Finally download the latest `.osm.pbf` from geofabrik [here](http://download.geofabrik.de/central-america/belize-latest.osm.pbf). We will call the folder that contains it `file:///path/to/pbf`.
+
+We will also trick the program into thinking that the 3 shards Belize intersects with this sharding tree each have a corresponding `.osm.pbf` file, which will each just be the same copy of the file that was downloaded from geofabrik:
```

Comment:
Perhaps missing "in" between with and this to read "intersects with in this sharding tree"